service/kafka.go

```diff
@@ -0,0 +1,30 @@
+package service
+
+import (
+	"context"
+
+	"github.com/ONSdigital/dp-cantabular-filter-flex-api/config"
+	kafka "github.com/ONSdigital/dp-kafka/v3"
+)
+
+// GetKafkaProducer creates a Kafka producer. Currently just for POST filters/{id}/submit
+func GetKafkaProducer(ctx context.Context, cfg *config.Config) (kafka.IProducer, error) {
+	pConfig := &kafka.ProducerConfig{
+		BrokerAddrs: cfg.KafkaConfig.Addr,
+		// TODO: Right default?
```

Comment:
can we remove this `TODO` or do we need to speak with someone else before this ticket is done?

Reply:
I think this was a hangover from earlier work on this branch. I believe the defaults are right. Removed.